oeps/processes/oep-0058-proc-translations-management.rst

```diff
@@ -0,0 +1,169 @@
+OEP-58: Using the GitHub Transifex App and a Separate Translation Repo as a Replacement for the edx-transifex-bot
```

Comment:
I think it'd be better to make this simple. The PR is titled "Translations Management" - how about that?

Reply:
I'm fine with that, but it's a bit vague. I'll make the Title be "Translations Management", and demote the title to subtitle.

Reply:
Feel free to suggest another, succinct title. Look at https://open-edx-proposals.readthedocs.io/en/latest/processes/oep-0001.html left sidebar, how other OEPs are titled. Yours would be a multi-line affair. Short & sweet is better.

Reply:
Agreed. Translations Management works for that, but there is value in having a subtitle that explains the short, sweet, and vague title a bit better. It won't show up in the left sidebar as a multi-line affair either.

Reply:
Removing subtitle, as its covered in the context section and messes up all the other headers.